components/automate-cli/cmd/chef-automate/cleanup.go

```diff
@@ -133,12 +135,25 @@ func runCleanupCmd(cmd *cobra.Command, args []string) error {
 				writer.Println("Reference architecture type :" + arch)
 
 				appendString := ""
-				if infra.Outputs.BackupConfigS3.Value == "true" && cleanupFlags.force {
-					appendString = appendString + `for i in 1;do i=$PWD;cd /hab/a2_deploy_workspace/terraform/destroy/aws/;cp -r ../../.tf_arch .;cp -r ../../../a2ha.rb ..;terraform apply -var="destroy_bucket=true" -auto-approve;cd $i;done`
+
+				backup_config, err := getTheValueFromA2HARB("backup_config")
+				if err != nil {
+					writer.Error("Error in getting backup_config")
+					return err
+				}
+
+				if backup_config == "s3" && cleanupFlags.force {
+					// bucket_name := config.GetGlobal().GetV1().GetBackups().GetS3().GetBucket().GetName().Value
```

Comment:
remove the commented code if not needed.

Reply:
Removed